build.edn

```diff
@@ -1,6 +1,8 @@
 {:lib toyokumo/toyokumo-commons
  :version "0.3.{{git/commit-count}}"
  :description "Utility functions that are used in multiple Toyokumo products"
+ :licenses [{:name "Apache License Version 2.0,"
```

Comment:
NIT The name does not have to be exact, so it is NIT, but there is a comma at the end.

Reply:
Fixed 38b35a3df3ddf22de10701114450fecd836a7119 😃